docs/docs/guides/dev-tools.md

```diff
@@ -15,14 +15,14 @@ Recoil has functionality to allow you to observe and update state changes.
 
 You can use a hook such as [**`useRecoilSnapshot()`**](/docs/api-reference/core/useRecoilSnapshot) or [**`useRecoilTransactionObserver_UNSTABLE()`**](/docs/api-reference/core/useRecoilTransactionObserver) to subscribe to state changes and obtain a [**`Snapshot`**](/docs/api-reference/core/Snapshot) of the new state.
 
-Once you have a `Snapshot`, you can use methods such as **`getLoadable()`**, **`getPromise()`**, and **`getInfo()`** to inspect the state and use **`getNodes()`** to iterate over the set of known atoms.
+Once you have a `Snapshot`, you can use methods such as **`getLoadable()`**, **`getPromise()`**, and **`getInfo()`** to inspect the state and use **`getNodes_UNSTABLE()`** to iterate over the set of known atoms.
```

Comment:
Ah yes, thanks!  Though, `getInfo()` should also be `getInfo_UNSTABLE()`  :)

Reply:
I haven't used `getInfo`. Thanks for reminding, it has been fixed